### PR TITLE
Add script for starting live version locally

### PR DESCRIPTION
**Pull request by rizwanmustafa**
This can help in testing live version locally and make live deployment easy for those who want to host locally

### CODE REVIEW

**Comment by Bruception** [comment]
Why is this needed?

File: frontend/package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "license": "GPL-3.0",
   "private": true,
   "scripts": {
+    "start": "yarn run build:live && cd public && npx serve -p 5000",
```
Comment (Bruception) [inline frontend/package.json:7]: Yarn? Serve? These may not be installed locally. I don’t really see the purpose of this PR.

Comment (rizwanmustafa) [reply frontend/package.json:7]: Sorry added yarn by mistake. Will change it with npm. The purpose is in the description. Having a one line script wont hurt the project anyways

Comment (rizwanmustafa) [reply frontend/package.json:7]: > Yarn? Serve? These may not be installed locally. I don’t really see the purpose of this PR.

But if you want I can close this PR

**Comment by Miodec** [comment]
I would rename script to `dev: live` or `live`

**Comment by Miodec** [comment]
> Why is this needed?

It could be useful to me in cases where the minified HTML is causing weird spacing issues in the design.

**Comment by Bruception** [comment]
> > Why is this needed?
> 
> It could be useful to me in cases where the minified HTML is causing weird spacing issues in the design.

How does this solve that?

File: frontend/package.json [hunk 2]
```diff
@@ -67,4 +68,4 @@
     "stemmer": "2.0.0",
     "throttle-debounce": "3.0.1"
   }
-}
+}
```
Comment (Bruception) [inline frontend/package.json:71]: Missing new line

File: frontend/package.json (hunk 1 above)
Comment (Bruception) [reply frontend/package.json:7]: > > Yarn? Serve? These may not be installed locally. I don’t really see the purpose of this PR.
> 
> But if you want I can close this PR

Its fine. Just resolve the issues. Don’t assume people have yarn/serve installed

**Comment by Miodec** [comment]
> > > Why is this needed?
> > 
> > 
> > It could be useful to me in cases where the minified HTML is causing weird spacing issues in the design.
> 
> How does this solve that?

This script will host the local server with a minified HTML. The HTML is not minified in the dev environment, which is causing some inconsistencies in spacing - most notably between links and text, because prettier puts them in new lines.

**Comment by rizwanmustafa** [comment]
@Miodec should I add serve to dev dependencies?

**Review by Miodec — APPROVED** [review]
Thanks

**Review by Miodec — APPROVED** [review]

**Pull request merged**